pages/buildkite_compute/pipeline_migration.md

```diff
@@ -0,0 +1,29 @@
+## Compute pipeline migration
+
+To migrate an existing pipeline to use a compute queue, you must first ensure:
+
+- Your pipeline is in the same cluster as all the compute queues you wish to target.
+- Each step in the pipeline targets the required compute queue.
+- Source control settings have been updated to allow code access.
+
+There are different pipeline repository settings required for public and private repositories, see below for the relevant instructions.
```

Comment:
Remove 'repository' from 'pipeline repository settings' since this setting relates to 'pipeline settings'.

```suggestion
Different pipeline settings are required for public and private repositories, see below for the relevant instructions.
```